__tests__/index.test.ts

```diff
@@ -1279,27 +1288,33 @@ describe('Auth0', () => {
         scope: TEST_SCOPES,
         client_id: TEST_CLIENT_ID
       });
-
-      expect(utils.getUniqueScopes).toHaveBeenCalledWith(
-        'openid profile email',
-        'openid profile email'
-      );
     });
 
     it('uses custom options when provided', async () => {
-      const { auth0, utils, cache } = await setup();
+      const { auth0, cache } = await setup();
       await auth0.getUser({ audience: 'the-audience', scope: 'the-scope' });
 
       expect(cache.get).toHaveBeenCalledWith({
         audience: 'the-audience',
-        scope: TEST_SCOPES,
+        scope: `${TEST_SCOPES} the-scope`,
         client_id: TEST_CLIENT_ID
       });
+    });
 
-      expect(utils.getUniqueScopes).toHaveBeenCalledWith(
```

Comment:
You should be able to do

```js
import * as scope from '../src/scope';

beforeEach(() => {
  jest.spyOn(scope, 'getUniqueScopes');
});

it('should do something', () => {
  // do something
  expect(scope.getUniqueScopes).toHaveBeenCalledWith(...);
})
```

https://jestjs.io/docs/en/jest-object#jestspyonobject-methodname
"Note: By default, `jest.spyOn` also calls the **spied** method. "

Reply:
I did try this but it wasn't working thanks to doing `jest.resetAllMocks()` as well. Turns out you need to [manually restore a spy](https://jestjs.io/docs/en/mock-function-api.html#mockfnmockrestore), as resetting a spy results in a function with no return value.

However, I don't believe the tests need to assert this any more as we're already asserting that the scopes are correct in assertions for `cache.get`, `utils.createQueryParams`, etc. I will leave the spy setup in though in case we want to mock it in the future and we forget about this problem ☝️